sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/UserDataTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using Xunit;
+
+namespace Compute.Tests
```

Comment:
Please include the generated json file made by running this test in Record mode.